app/api/models/domain/Nino.scala

```diff
@@ -17,11 +17,9 @@
 package api.models.domain
 
 case class Nino(nino: String) {
-  require(Nino.isValid(nino), s"$nino is not a valid nino.")
   private val LengthWithoutSuffix = 8
 
-  val value: String = nino
-  val name          = "nino"
+  def value: String = nino
```

Comment:
Can delete this if the "def formatted" and "def withoutSuffix" below both use "nino" instead of "value"